cmd/lakectl/cmd/merge.go

```diff
@@ -34,6 +34,7 @@ var mergeCmd = &cobra.Command{
 		return validRepositoryToComplete(cmd.Context(), toComplete)
 	},
 	Run: func(cmd *cobra.Command, args []string) {
+		message, kvPairs := getCommitFlags(cmd)
```

Comment:
Does `allow-empty-message` fit here? What's the default behavior that we now change?

Reply:
Fixed to fallback to previous default behavior